app/Http/Controllers/UserController.php

```diff
@@ -12,10 +14,27 @@ class UserController extends Controller
      *
      * @return \Illuminate\Http\Response
      */
-    public function index()
+    public function index(Request $request)
     {
         //
-    }
+        $users = [];
```

Comment:
There's no need for this line

Reply:
the line ensures if db connection was not successful, an empty array is returned instead of error

Reply:
Well, Laravel returns an empty array by default. You just have to check if the value is set in your blade files.

Reply:
That will be the case if it's not in the try-catch block unless you're proposing we should not catch such exceptions.